pkspace/utils/loaders.py

```diff
@@ -0,0 +1,105 @@
+import glob
+import json
+import os
+import xml.etree.ElementTree
+import cv2
+import numpy as np
+
+
+class Loader:
+    def load(self, path, split):
+        pass
+
+
+class PKspaceLoader(Loader):
+    def __get_spaces__(self, src, desc):
+        finsize = (80, 80)
+        spaces = []
+        answers = []
+        for space in desc.get('spots'):
+            points = space.get('points')
+            angle = space.get('rotation')
+            nove_pole = np.hstack(
+                (np.asarray(points), np.ones((len(points), 1))))
+            center = np.mean(nove_pole, 0)[:2]
+            center = (center[0], center[1])
+            rows, cols, tmp = src.shape
+            m = cv2.getRotationMatrix2D(center, -angle, 1)
```

Comment:
@rstevanak In order to make the code more readable, I suggest you use whitespace to separate different logical blocks. This may be a good line to do that.